docs/pages/upgrading.mdx

```diff
@@ -1,47 +1,22 @@
 ---
 title: Upgrading Teleport
 description: Explains how to upgrade Teleport depending on your environment and edition.
-layout: tocless-doc
 ---
 
 The guides in this section show you how to upgrade Teleport to a more recent
 version.
 
-## Upgrading compatibility overview
+Read the [Upgrading Compatibility Overview](./upgrading/overview.mdx) to
+understand how to upgrade components in your Teleport cluster while ensuring
+compatibility between all components.
 
-Since Teleport is a distributed system with a number of services that run on
-potentially many hosts, you should take care when upgrading the cluster to
-ensure that all components remain compatible.
+If you have a Teleport Enterprise (Cloud) account, you can [set up automatic
+Teleport agent updates](./upgrading/automatic-agent-updates.mdx) to ensure that
+the version of Teleport running on agents is always compatible with that of the
+Teleport cluster.
```

Comment:
Can we make the language stronger here? Automatic upgrades is a requirement, not a suggestion.